Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +493,125 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         return [delete]
     }
 }
+
+class SyncedTabsTableViewCell: UIView {
+    var navigationController: UINavigationController? = nil
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    var profile: Profile!
+    weak var historyPanelController: HistoryPanelViewController!
+
+    private let ImageMargin: CGFloat = 12
+
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = HistoryPanelUX.SyncedTabsCellTextColor
+        label.text = Strings.SyncedTabsTableViewCellTitle
+        label.font = DynamicFontHelper.defaultHelper.DeviceFontHistoryPanel
+        return label
+    }()
+
+    lazy var descriptionLabel: UILabel? = {
+        return nil
+    }()
+
+    lazy var imageView: UIImageView = {
+        let image = UIImage(named: "panelIconSyncedTabs")!
+        return UIImageView(image: image)
+    }()
+
+    lazy var chevron: ChevronView = {
+        let chevron = ChevronView(direction: .Right)
+        chevron.tintColor = HistoryPanelUX.SyncedTabsCellTextColor
+        chevron.lineWidth = 3.0
+        return chevron
+    }()
+
+    lazy var topBorder: UIView = {
+        let view = UIView()
+        view.backgroundColor = SiteTableViewControllerUX.HeaderBorderColor
+        return view
+    }()
+
+    lazy var bottomBorder: UIView = {
+        let view = UIView()
+        view.backgroundColor = SiteTableViewControllerUX.HeaderBorderColor
+        return view
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        userInteractionEnabled = true
+        backgroundColor = SiteTableViewControllerUX.HeaderBackgroundColor
+
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(SyncedTabsTableViewCell.viewWasTapped(_:)))
+        tapGestureRecognizer.numberOfTapsRequired = 1
+        addGestureRecognizer(tapGestureRecognizer)
+
+        addSubview(topBorder)
+        addSubview(bottomBorder)
+        addSubview(chevron)
+        addSubview(titleLabel)
+        addSubview(imageView)
+
+        imageView.snp_makeConstraints { make in
+            make.left.equalTo(self).offset(TwoLineCellUX.BorderViewMargin + (0.5 * TwoLineCellUX.BorderViewMargin))
+            make.centerY.equalTo(self)
+            make.width.equalTo(HistoryPanelUX.SyncedTabsCellImageSize)
+        }
+
+        chevron.snp_makeConstraints { make in
+            make.right.equalTo(self).offset(-HistoryPanelUX.SyncedTabsCellChevronInset)
+            make.centerY.equalTo(self)
+            make.size.equalTo(HistoryPanelUX.SyncedTabsCellChevronSize)
+        }
+
+        topBorder.snp_makeConstraints { make in
+            make.left.right.equalTo(self)
+            make.top.equalTo(self).offset(-0.5)
+            make.height.equalTo(0.5)
+        }
+
+        bottomBorder.snp_makeConstraints { make in
+            make.left.right.bottom.equalTo(self)
+            make.height.equalTo(0.5)
+        }
+
+        updateConstraints()
```

Comment:
Once you keep the `descriptionLabel` around, you shouldn't need this `updateConstraints` call or anything in the overridden `updateConstraints` method below.